canonical-websites/docs.ubuntu.com
# Update to Vanilla v1.8

## Done
- Updated version of site to v1.8
- Updated sub-heading in hero to `h4` and removed is-deep class
- Fixed navigation logo
- Add favicon to site

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at: [http://0.0.0.0:8007](http://0.0.0.0:8007)
4. Compare with [live](https://docs.ubuntu.com/) and ensure that the site looks the same (or better)
